frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx

```diff
@@ -61,7 +63,7 @@ export const PropertyDetailsTabView: React.FunctionComponent<IPropertyDetailsTab
       <StyledSummarySection>
         <LoadingBackdrop show={loading} parentScreen={true} />
         <StyledEditWrapper className="mr-3 my-1">
-          {setEditMode !== undefined && (
+          {setEditMode !== undefined && hasClaim(Claims.PROPERTY_EDIT) && (
```

Comment:
the edit property icon should be controlled by claims

Reply:
Was this reported by a PROD user that warrants a hotfix? I was about to make the same change on a related bug fix in TEST

Reply:
yes, maybe a duplicate report in uat/test